manifests/plugin/dynflow.pp

```diff
@@ -32,4 +32,18 @@
     listen_on     => $listen_on,
     template_path => 'foreman_proxy/plugin/dynflow.yml.erb',
   }
+
+  if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
+    $scl_prefix = 'tfm-'
+  } else {
+    $scl_prefix = ''
+  }
+
+  foreman_proxy::plugin { 'dynflow_core':
+    package => "${scl_prefix}${::foreman_proxy::plugin_prefix}dynflow_core",
+  } ->
```

Comment:
Maybe notify it? ```~>```. The package can be upgraded, but if the RPM does a conditional restart this it may not be needed.